README.md

```diff
@@ -76,3 +76,38 @@ $ docker run --tty \
+
+
+
+Next step is to launch the AWS batch job(s) to generate the tiles through the uploaded docker image. Tiles are generated for a _service area_ defined in the source code: [generate_tiles.py](https://github.com/mbta/tile-server/blob/master/etc/generate_tiles.py#L195-L198). This area roughtly represents the area, which customers are believed to expect to see on the maps on the mbta.com website. If you change the area in the source code -- or do any other change there -- you need to rebuild the docker image and push it to the ECR repo (see above).
+
+The idea behind running the jobs in AWS batch is that we split the _service area_ into horizontal (East-West) stripes and generate tiles for each of those map stripes independently. Since there is no overlap, the execution can be parallelized and we can have as many parallel jobs as we see necessary. 
+
+Important thing to note is that the service area only make sense if we have the map data for it. Currently we have map data for MA, RI and NH, so if you need service area to cover any other territories, you need to add map data first ([example PR](https://github.com/mbta/tile-server/pull/12/files)).
+
+In order to launch AWS Batch jobs, do the following:
+* Login into AWS Console and navigate to AWS Batch: https://console.aws.amazon.com/batch/
+* Go to "Job Definitions" section, click on "tile-generation-dev" item and then on the radio-button corresponding to the latest revision. 
+* Click Actions -> Submit job
+* On the next screen do the following:
+  * Specify job name, which describes current job the best (no special requirements for the name)
+  * Specify `tile-generation-dev-queue` as the job queue name. 
+  * **Important**: specify **Array** as job type. 
+  * Specify number of jobs to be run as parallel in Array Size field. If you don't know how many you need, put `8`. It took 5-6 hours for 8 jobs to generate the tiles for entire service area up to maximum zoom level (18 during previous tests. 
+  * Navigate to **Environment variables** section and make sure that we have both mandatory variables set to correct values:
+    * `MAPNIK_TILE_S3_BUCKET`: should correspond with the target S3 bucket; `mbta-map-tiles-dev` for dev environment or `mbta-map-tiles` for production environment. If you want to upload files to different bucket, you can specify it here as well, but make sure that batch jobs have permissions to write there. 
+    * **Important**: `BATCH_JOB_COUNT`: should be equal to what you specified in Array Size field. Failure to set the right value here will result in either missing tiles or in overal between the jobs (they are going to generate the same tiles).
+  * Click "Submit Job" and wait for jobs to run. You can also check Cloud Watch logs, while the jobs are running. Links to those logs are available from the Jobs page. 
+
+These steps should also normally be set up to run in CI system, but at the time of writing this we don't have one, so they need to be executed manually. 
```

Comment:
is there a ticket to do this? (possibly part of the same CI as the Docker file build)

Reply:
I think it's the same @ianwestcott's task - https://app.asana.com/0/1113179098808463/1116455779572731/f

Reply:
It's actually this task, which was recently completed as Won't Do: https://app.asana.com/0/947623132087374/1117279835834479 Feel free to reopen it and advocate for it if you want to see it done.

In the meantime, we could write a shell script to encapsulate the logic in anticipation of it being run by automation.

Reply:
I still suggest that it's the task I linked - https://app.asana.com/0/947623132087374/1116455779572731/f - its title ("Set up periodic task for tile-server image generation/publishing") refers to the that same activity as I'm describing here. 

I don't think there any doubts about whether this is required, or are there, @gshkolnik? If we don't do it our maps are going to become outdated quickly.

Reply:
Sorry, I might have replied to the wrong comment. But I gather there are two distinct tasks at issue regarding automation:

1. [Building and publishing the container via CI](https://app.asana.com/0/947623132087374/1117279835834479)
1. [Running tile generation periodically](https://app.asana.com/0/947623132087374/1116455779572731/f)

The latter is in my backlog. The former was marked "Won't Do" but IMO we can add a script to the repo in lieu of creating a Semaphore project.